lighthouse-cli/test/smokehouse/test-definitions/perf/expectations.js

```diff
@@ -199,4 +199,35 @@ module.exports = [
       },
     },
   },
+  {
+    lhr: {
+      requestedUrl: 'http://localhost:10200/perf/trace-elements.html?missing',
+      finalUrl: 'http://localhost:10200/perf/trace-elements.html?missing',
+      audits: {
+        'largest-contentful-paint-element': {
+          score: null,
+          displayValue: '1 element found',
```

Comment:
please remove display values, using an array in `details.items` is already asserting the length.